.github/workflows/spelling2.yml

```diff
@@ -43,12 +66,13 @@ on:
   pull_request_target:
     branches:
     - "**"
-    tags-ignore:
-    - "**"
     types:
     - 'opened'
     - 'reopened'
     - 'synchronize'
+  issue_comment:
+    types:
+    - 'created'
```

Comment:
This enables users to talk to check-spelling (but it's limited below to only run on forks)